src/chi/components/checkbox/checkbox.scss

```diff
@@ -72,8 +72,13 @@
     &:checked {
       & + .chi-checkbox__label {
         &::before {
-          background-color: $color-background-primary;
-          border-color: $color-border-primary;
+          @if $theme == ctl {
+            background-color: $ctl-color-background-primary;
+            border-color: $ctl-color-border-primary;
+          } @else {
+            background-color: $color-background-primary;
+            border-color: $color-border-primary;
+          }
```

Comment:
I would recommend avoiding if theme statements at the component level as we will eventually support multiple themes and this method would not be sustainable. Instead of changing tokens/variable names at the component level, I would suggest doing all theming in the variable files. Each theme would have its own variables file containing the same variable names but different values.